river/command/rule.zig

```diff
@@ -107,6 +120,13 @@ pub fn ruleDel(_: *Seat, args: []const [:0]const u8, _: *?[]const u8) Error!void
             apply_ssd_rules();
             server.root.applyPending();
         },
+        .tag => {
+            server.config.tag_rules.del(.{
+                .app_id_glob = app_id_glob,
+                .title_glob = title_glob,
+                .value = undefined, // Will not be accessed.
```

Comment:
I think this use of `undefined` is a strong indication that the `del()` function's API should be changed to take a `struct { app_id_glob: []const u8, title_glob: []const u8 }` instead.